### ISSUE TITLE
Update build documentation with fix for dual ABI

### ISSUE THREAD

**Issue by andreamattavelli** [klee/klee.github.io#50]
We should update documentation accordingly according to @delcypher 's comment in https://github.com/klee/klee/issues/336#issuecomment-181827009.

**Comment by 0152la** [comment]
Wanted to share my experience with KLEE, LLVM and ABI. I'm on Ubuntu Gnome 17.04, with g++ 6.3.0, and built LLVM 3.4 from source using the Makefile, with `--enable-optimized`. When building KLEE, I got the ABI issues mentioned. When adding the cmake flag to disable ABI, I saw tons of undefined references. I had to go back and change the LLVM Makefile to also disable ABI in order to be able to build KLEE, by adding `-D_GLIBCXX_USE_CXX11_ABI=0` to CXXFLAGS (line 166 of `Makefile.config`).

I fairly recently did the same on a Ubuntu 16.04 with g++ 5.4.0 and did not encounter any issue with ABI during the building process.
